<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio</class>
 <widget class="QMainWindow" name="inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>inicio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Imágenes/images.png</normaloff>../../../../Imágenes/images.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>576</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/world-of-warships-legends-llega-a-la-consola-con-acceso-temprano-en-todo-el-mundo-frikigamers.com_.jpg-1024x576.jpg</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>273</width>
       <height>134</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tamaño del Mapa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSlider_Size">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Cantidad de Barcos</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="horizontalSlider_Ships_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox_2"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="Buton_ADD_Date">
        <property name="text">
         <string>Cargar datos</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="Line_Name_Input"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>258</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="Check_Box_Pos_Random">
        <property name="text">
         <string>Posicionamiento aleatorio</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Barcos Disponibles</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="BoxShips">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Portaaviones</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Destructor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crucero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Submarino</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lancha de Combate</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Coordenada X:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LineCoorY"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Coordenada Y:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="LineCoorX"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Orientacion:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="BoxOrientacion">
        <item>
         <property name="text">
          <string>Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string notr="true">%p%</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="Buton_ADD_Ship">
        <property name="text">
         <string>Cargar Barco</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="Check_Box_Ships_Random">
        <property name="text">
         <string>Barcos aleatorios</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>480</y>
       <width>150</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ButonNewGame">
        <property name="styleSheet">
         <string notr="true">  min-width: 130px;
  height: 40px;
  color: #fff;
  padding: 5px 10px;
  font-weight: bold;
  cursor: pointer;
  transition: all 0.3s ease;
  position: relative;
  display: inline-block;
  outline: none;
  border-radius: 20px;
  border: 2px solid #212529;
  background: #212529;</string>
        </property>
        <property name="text">
         <string>Nuevo Juego</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButonLoadGame">
        <property name="styleSheet">
         <string notr="true">  min-width: 130px;
  height: 40px;
  color: #fff;
  padding: 5px 10px;
  font-weight: bold;
  cursor: pointer;
  transition: all 0.3s ease;
  position: relative;
  display: inline-block;
  outline: none;
  border-radius: 20px;
  border: 2px solid #212529;
  background: #212529;</string>
        </property>
        <property name="text">
         <string>Cargar Partida</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>311</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QPushButton" name="start_combat">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>43</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  min-width: 130px;
  height: 40px;
  color: #fff;
  padding: 5px 10px;
  font-weight: bold;
  cursor: pointer;
  transition: all 0.3s ease;
  position: relative;
  display: inline-block;
  outline: none;
  border-radius: 20px;
  border: 2px solid #212529;
  background: #212529;</string>
      </property>
      <property name="text">
       <string>COMENZAR BATALLA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tablero" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1024</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>521</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0" rowminimumheight="0" columnminimumwidth="0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>10</y>
         <width>481</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="bottomMargin">
         <number>3</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>300</y>
        <width>120</width>
        <height>64</height>
       </rect>
      </property>
      <widget class="QPushButton" name="ButonSave">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>310</y>
        <width>171</width>
        <height>125</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>310</y>
        <width>171</width>
        <height>125</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="LabelDef">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/ImgDefeatRev.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="LabelWinn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/ImgVictoriRev.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>154</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  min-width: 130px;
  height: 40px;
  color: #fff;
  padding: 5px 10px;
  font-weight: bold;
  cursor: pointer;
  transition: all 0.3s ease;
  position: relative;
  display: inline-block;
  outline: none;
  border-radius: 20px;
  border: 2px solid #212529;
  background: #212529;</string>
     </property>
     <property name="text">
      <string>Continuar</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>850</width>
       <height>430</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>850</width>
       <height>430</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>850</width>
       <height>430</height>
      </size>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="LayoutEsp1"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>0</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="LayoutEsp2"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>581</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutEsp1"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>310</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutEsp2"/>
     </widget>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>560</y>
       <width>154</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>154</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">  min-width: 130px;
  height: 40px;
  color: #fff;
  padding: 5px 10px;
  font-weight: bold;
  cursor: pointer;
  transition: all 0.3s ease;
  position: relative;
  display: inline-block;
  outline: none;
  border-radius: 20px;
  border: 2px solid #212529;
  background: #212529;</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
